README.md

```diff
@@ -145,6 +145,16 @@ Variable | Description | Example
 **NOTE**: The mobile apps won't work with self-signed certificates (the default)
 see below for instructions on how to obtain a proper certificate with Let's Encrypt.
 
+**Let's Encrypt rate limit warning**: Let's Encrypt has a limit to how many times you can submit a request
```

Comment:
Can you please move this to the section below, under the Let's Encrypt configuration title?